<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VSCCamInfo</class>
 <widget class="QWidget" name="VSCCamInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>591</width>
    <height>369</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Master Hard Disk Selecting</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/logo/resources/vscsmall.jpg</normaloff>:/logo/resources/vscsmall.jpg</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QGridLayout" name="gridLayout_4">
       <property name="horizontalSpacing">
        <number>30</number>
       </property>
       <property name="verticalSpacing">
        <number>20</number>
       </property>
       <item row="0" column="2">
        <widget class="QRadioButton" name="radioButtonRtsp">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">QRadioButton::indicator {
    width: 20px;
    height: 20px;
}

QRadioButton::indicator::unchecked {
    image: url(:/action/resources/radiobutton_unchecked.png);
}

QRadioButton::indicator:unchecked:hover {
    image: url(:/action/resources/radiobutton_unchecked_hover.png);
}

QRadioButton::indicator:unchecked:pressed {
    image: url(:/action/resources/radiobutton_unchecked_pressed.png);
}

QRadioButton::indicator::checked {
    image: url(:/action/resources/radiobutton_checked.png);
}

QRadioButton::indicator:checked:hover {
    image: url(:/action/resources/radiobutton_checked_hover.png);
}

QRadioButton::indicator:checked:pressed {
    image: url(:/action/resources/radiobutton_checked_pressed.png);
}</string>
         </property>
         <property name="text">
          <string>RTSP</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QRadioButton" name="radioButtonOnvif">
         <property name="styleSheet">
          <string notr="true">QRadioButton::indicator {
    width: 20px;
    height: 20px;
}

QRadioButton::indicator::unchecked {
    image: url(:/action/resources/radiobutton_unchecked.png);
}

QRadioButton::indicator:unchecked:hover {
    image: url(:/action/resources/radiobutton_unchecked_hover.png);
}

QRadioButton::indicator:unchecked:pressed {
    image: url(:/action/resources/radiobutton_unchecked_pressed.png);
}

QRadioButton::indicator::checked {
    image: url(:/action/resources/radiobutton_checked.png);
}

QRadioButton::indicator:checked:hover {
    image: url(:/action/resources/radiobutton_checked_hover.png);
}

QRadioButton::indicator:checked:pressed {
    image: url(:/action/resources/radiobutton_checked_pressed.png);
}</string>
         </property>
         <property name="text">
          <string>ONVIF</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QRadioButton" name="radioButtonFile">
         <property name="styleSheet">
          <string notr="true">QRadioButton::indicator {
    width: 20px;
    height: 20px;
}

QRadioButton::indicator::unchecked {
    image: url(:/action/resources/radiobutton_unchecked.png);
}

QRadioButton::indicator:unchecked:hover {
    image: url(:/action/resources/radiobutton_unchecked_hover.png);
}

QRadioButton::indicator:unchecked:pressed {
    image: url(:/action/resources/radiobutton_unchecked_pressed.png);
}

QRadioButton::indicator::checked {
    image: url(:/action/resources/radiobutton_checked.png);
}

QRadioButton::indicator:checked:hover {
    image: url(:/action/resources/radiobutton_checked_hover.png);
}

QRadioButton::indicator:checked:pressed {
    image: url(:/action/resources/radiobutton_checked_pressed.png);
}</string>
         </property>
         <property name="text">
          <string>FILE</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QRadioButton" name="radioButtonMJPEG">
         <property name="styleSheet">
          <string notr="true">QRadioButton::indicator {
    width: 20px;
    height: 20px;
}

QRadioButton::indicator::unchecked {
    image: url(:/action/resources/radiobutton_unchecked.png);
}

QRadioButton::indicator:unchecked:hover {
    image: url(:/action/resources/radiobutton_unchecked_hover.png);
}

QRadioButton::indicator:unchecked:pressed {
    image: url(:/action/resources/radiobutton_unchecked_pressed.png);
}

QRadioButton::indicator::checked {
    image: url(:/action/resources/radiobutton_checked.png);
}

QRadioButton::indicator:checked:hover {
    image: url(:/action/resources/radiobutton_checked_hover.png);
}

QRadioButton::indicator:checked:pressed {
    image: url(:/action/resources/radiobutton_checked_pressed.png);
}</string>
         </property>
         <property name="text">
          <string>MJPEG</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>10</number>
       </property>
       <item row="4" column="0">
        <widget class="QLabel" name="labelIP_2">
         <property name="text">
          <string>IP Address:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="labelName_4">
         <property name="text">
          <string>File Name:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QLineEdit" name="lineEditFile">
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="lineEditPort">
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="labelName_3">
         <property name="text">
          <string>RTSP Address:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLineEdit" name="lineEditRtspAddr">
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="labelName">
         <property name="text">
          <string>Camera Name:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="labelPort_2">
         <property name="text">
          <string>Port:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="lineEditIP">
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="lineEditName">
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../main/vscmainwindows.qrc"/>
 </resources>
 <connections/>
</ui>
